doc/nrf/ug_tfm.rst

```diff
@@ -18,6 +18,8 @@ It provides a highly configurable set of software components to create a Trusted
 This is achieved by a set of secure run time services such as Secure Storage, Cryptography, Audit Logs, and Attestation.
 Additionally, secure boot via MCUboot in TF-M ensures integrity of run time software and supports firmware upgrade.
 
+Support for TF-M in |NCS| is currently experimental, however it is more robust and complete software offering which will be the default option in the future.
```

Comment:
@b-gent you probably need to do this here as well

Reply:
I mean, the same comment update...

Reply:
Thanks, should be fine now.